mxenabled/connect-widget
# Always show back button when interstitial Disclosure is rendered

Issue: [https://mxcom.atlassian.net/browse/CT-1061](https://mxcom.atlassian.net/browse/CT-1061)

### Changes
Allows show back button when interstitial disclosure is rendered and `disable_institution_search` is true.


### Testing instructions

- Ensure the back button is showing when interstitial disclosure is rendered and `disable_institution_search` is true.
- Run both Vitest and Cypress tests
